tests/hlsl-intrinsic/shader-execution-reordering/hit-object-trace-motion-ray.slang.1.expected

```diff
@@ -0,0 +1,88 @@
+result code = 0
+standard error = {
+}
+standard output = {
+#version 460
+#extension GL_EXT_ray_tracing : require
+#extension GLSL_NV_ray_tracing_motion_blur : require
+#extension GL_NV_shader_invocation_reorder : require
+layout(row_major) uniform;
+layout(row_major) buffer;
+struct SomeValues_0
+{
+    int a_0;
+    float b_0;
+};
+
+layout(location = 0)
+rayPayloadEXT
+SomeValues_0 p_0;
+
+layout(binding = 0)
+uniform accelerationStructureEXT scene_0;
+
+layout(location = 0)
+hitObjectAttributeNV
+SomeValues_0 t_0;
+
+SomeValues_0 HitObject_GetAttributes_0(hitObjectNV this_0)
+{
+    return t_0;
```

Comment:
This should be extracting attributes from hitObjectNV using
`hitObjectGetAttributesNV(this_0, 0)`
But I think you have mentioned this as TODO/WIP

I am seeing sensible codegen from our stack (before entering rtcore) for most of the examples(havent checked all) so change looks good to me